src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
@davidsh @CIPop , this modificaiton is because IsNotWindowsImplementation was returning true for Nano. Note that IsNotOneCoreUAP looks for the presence of httpapi.dll which apparenlty is not available on Nano. Can you confirm this change is correct? Or does the absence of httpapi.dll on Nano mean that you use the managed implementation, so this change is probably not correct -- the property should be renamed IsNativeWindowsImplementation for clarity.?